apps/services/documents/src/app/modules/document-provider/controllers/changelog.controller.ts

```diff
@@ -1,12 +1,12 @@
 import { IdsUserGuard, Scopes, ScopesGuard } from '@island.is/auth-nest-tools'
-import { ApiScope } from '@island.is/auth/scopes'
+import { AdminPortalScope } from '@island.is/auth/scopes'
 import { Controller, Get, Param, UseGuards } from '@nestjs/common'
 import { ApiHeader, ApiOkResponse, ApiTags } from '@nestjs/swagger'
 import { DocumentProviderService } from '../document-provider.service'
 import { Changelog } from '../models/changelog.model'
 
 @UseGuards(IdsUserGuard, ScopesGuard)
-@Scopes(ApiScope.internal)
+@Scopes(AdminPortalScope.documentProvider)
```

Comment:
After taking a second look I saw that CreateOrganisation in the OrganisationController is also being called directly from document-provider-onboarding.service so I added internal scope to that route, I could not find another reference to the controller, but this will require more tests once it is out on dev where all the services are up and running.